static/app/components/events/interfaces/frame/utils.tsx

```diff
@@ -138,3 +138,11 @@ function isAnrEvent(event: Event) {
     mechanismTag === 'mx_hang_diagnostic';
   return isANR;
 }
+
+export function hasFileExtension(filepath: string) {
+  // Regular expression to match a file extension
+  const fileExtensionPattern = /\.[0-9a-z]+$/i;
```

Comment:
is this just checking for a period anywhere in the name? that might not be great